integration.js

```diff
@@ -128,6 +228,91 @@ function getDetects(detectIds, options, cb) {
   });
 }
 
+function onDetails(lookupObject, options, cb) {
+  async.waterfall(
+    [
+      (cb) => {
+        if (
+          lookupObject &&
+          lookupObject.data &&
+          lookupObject.data.details &&
+          lookupObject.data.details.detectIds &&
+          lookupObject.data.details.detectIds.data &&
+          lookupObject.data.details.detectIds.data.details &&
+          lookupObject.data.details.detectIds.data.details.detectIds &&
+          lookupObject.data.details.detectIds.data !== null
```

Comment:
@ntsmith7 I don't think this null check at the end here is doing anything since we already check that data exists further up in the chain.  I noticed that lodash is being used in the `package.json`.  It might be worth just using the `get` method from lodash here:

```
const detectIds = _.get(lookupObject, 'data.details.detectIds.data.details.detectIds', null);
if(detectIds !== null){
   // continue on
}
```